draft-wdenniss-oauth-device-posture.xml

```diff
@@ -114,25 +133,61 @@
         follows:
       </t>
       <t>
-        <list style="hanging"> 
+        <list style="hanging">
           <t hangText="screen_lock"><vspace/>
-          Boolean.  True if the user has a screen lock, such as a pin, pattern
+          Boolean. True if the device has a screen lock, such as a pin, pattern
           biometric, etc.
-          </t> 
+          </t>
 
           <t hangText="root_privileges"><vspace/>
           Boolean. True if user apps can access root device privileges. For mobile
           operating systems, known as "jailbreaking" on iOS and "rooting" on
           Android.
           </t>
 
+          <t hangText="full_disk_encryption"><vspace/>
+          Boolean. True if data stored on the device is fully encrypted at rest.
+          </t>
+
+          <t hangText="device_id"><vspace/>
+          String. A unique identifier for the device.
+          </t>
+
+          <t hangText="device_os"><vspace/>
+          String. The name of the operating system running on the device such as
+          "iOS" or "Android".
+          </t>
+
+          <t hangText="device_os_version"><vspace/>
+          String. The current version of the operating system.
+          </t>
+
+          <t hangText="device_vendor"><vspace/>
+          String. The vendor of the device such as "Apple" or "Google".
+          </t>
+
+          <t hangText="device_model"><vspace/>
+          String. The model of the device such as "iPhone X" or "Pixel 2".
+          </t>
+
           <t hangText="device_attestation"> <vspace/>
           Dictionary. An attestation from the operating system, containing
-          a signed-statement about the device and/or the app. The format is 
+          a signed-statement about the device and/or the app. The format is
           a dictionary, the specifics of which depends on the operating system.
           </t>
-        </list> 
-      </t> 
+
+          <t hangText="app_id"><vspace/>
```

Comment:
This was taken from FIDO https://fidoalliance.org/specs/fido-u2f-v1.0-ps-20141009/fido-appid-and-facets-ps-20141009.html#the-appid-and-facetid-assertions as John suggested during our discussion at Oktane